app/models/notification_type.rb

```diff
@@ -20,7 +20,11 @@ def subscriber_ids(subject, initiator)
     when AUDIENCE_GROUP
       subject.try(:requester).try(:current_group).try(:user_ids)
     when AUDIENCE_TENANT
-      subject.tenant.user_ids
+      if subject.respond_to? :tenant
+        subject.tenant
+      elsif initiator.kind_of? User
+        initiator.current_group.try(:tenant)
```

Comment:
@isimluk I think you can just call `current_tenant` on the user instance here  because it's delegated to the `MiqGroup` class in the `User` class.